addons/event_sale/tests/test_event_sale.py

```diff
@@ -394,6 +394,7 @@ def test_ticket_price_with_pricelist_and_tax(self):
         self.env.user.partner_id.country_id = False
         pricelist = self.env['product.pricelist'].create({'name': 'Base Pricelist'})
 
+        self.env['account.tax.group'].search([]).country_id = self.env.company.account_fiscal_country_id
```

Comment:
This belongs probably to a setup, here it is a bit lost in the middle of a test without any explanation. Why do we need this, what is its purpose,... ? As we are in a side addon, better be explicit about it and help other people adding and improving tests :) .

Reply:
Put in the setup of `event` (`EventCase`)